perun-cli/getAttribute

```diff
@@ -116,7 +118,45 @@ if (defined $entitylessSubj) {
 	defined ($uesId) and $parameters{userExtSource} = $uesId;
 }
 
-my @attributes = $attributesAgent->getAttribute( %parameters );
+# find if exists combination of entered parameters
+my $is=0;
+if (exists($parameters{'resource'}) and exists($parameters{'member'})) {
+	my $resourcesAgent = $agent->getResourcesAgent;
+	my @members = $resourcesAgent->getAllowedMembers( resource => $resourceId );		             	
```

Comment:
I would prefer getting assigned members and not allowed (as you state on your own in error message ``member is not assigned to resource``). Allowed means member is also valid, but assigned represents connection between them.